docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -476,6 +476,58 @@ This behavior was confirmed for a number of existing tools:
 [poetry](https://github.com/python-poetry/poetry/blob/1c04c65149776ae4993fa508bef53373f45c66eb/src/poetry/utils/wheel.py#L23-L27),
 [uv](https://github.com/astral-sh/uv/blob/f6a9b55eb73be4f1fb9831362a192cdd8312ab96/crates/uv-distribution-filename/src/wheel.rs#L182-L299).
 
+### Variant properties system
+
+Variant properties follow a key-value design, where namespace and feature name constitute the key. Namespaces are used
+to group features defined by a single provider, and avoid conflicts should multiple providers define a feature with the
+same name. These keys are restricted to lowercase letters, digits and underscores, to make it easier to preserve
+consistency between different providers. In particular, uppercase characters are disallowed to avoid different spellings
```

Comment:
> to make it easier to preserve consistency between different providers.

IMO we can cut this part out, it's a common practice for PEPs to restrict character ranges, and we aren't providing a rationale for this particular range (which is fine, we only need to pick some reasonable range).